### ISSUE TITLE
Connecting to ollama 

### ISSUE THREAD

**Issue by govindkailas** [lee88688/aider-composer#38]
How do I connect to `ollama` running locally? I tried the below from the settings, 
![image](https://github.com/user-attachments/assets/ba694242-d451-4cb5-8432-a22812297402)

But it doesn't seem to be working 

![image](https://github.com/user-attachments/assets/a85cdaa8-07a6-43e9-9bf5-d0adcd239ab8)

I see the spinning wheel for a while, but I don't receive any response.

Also, when I go back to the settings, everything is reset. It doesn't store the details of the `ollama` I provided earlier. 

![image](https://github.com/user-attachments/assets/5fe9c1db-72b7-4d31-ba82-4880090d874d)

It would be helpful if you could include additional documentation on this topic.

**Comment by lee88688** [comment]
Did you click save?

**Comment by govindkailas** [comment]
> Did you click save?

yes, I did. I also tried with `OpenAI Compactible`, but it's the same.

**Comment by govindkailas** [comment]
I am connected to a remote host via ssh (vscode remote development), if that helps. But no logs or warnings. 
It issued a warning while I was setting the Python path; I expected a similar message if it wasn't connecting to Ollama.

**Comment by lee88688** [comment]
> I am connected to a remote host via ssh (vscode remote development), if that helps. But no logs or warnings. It issued a warning while I was setting the Python path; I expected a similar message if it wasn't connecting to Ollama.

the extension currently not supported vscode remove. when start, extension will start a aider service in the host(local or remote). but when in remote, webview in vscode(the chat and setting page) cannot connect host aider server.
maybe there is an solution for automatically redirect remote port to local.

**Comment by govindkailas** [comment]
It would be nice to at least show some warning if the connectivity is failing; just like the Python path settings. 

While trying OpenAI Compactible, got the below, 

console logs from vscode, 
![image](https://github.com/user-attachments/assets/830ca1fa-eb53-49cc-871d-0234008917ac)

**Comment by lee88688** [comment]
I am not sure if the official has provided relevant APIs for forwarding the host's port. so that I can use to auto redirect the aider's server to local. I may need to search it.

**Comment by lee88688** [comment]
if you use SSH remote, the config(path path) need to be set to remote server's pathon path.

**Comment by govindkailas** [comment]
A huge shoutout to @lee88688  for taking the time to consider this request! Your support is going to make a real difference for so many people. Thank you! 💯
